schema

```diff
@@ -0,0 +1 @@
+./python/lsst/alert/packet/schema
```

Comment:
It feels weird to me to have the schema files themselves buried in the `python` directory. Not dealbreaker-weird, but is it really necessary?

Reply:
See https://github.com/lsst/alert_packet/pull/15/commits/4ae3eebc0167d45073b4abc4cdfb5c2af9a7b2f5 - the docs on "how to add data files that aren't part of a python package" literally just respond with "Instead, the PyPA recommends that any data files you wish to be  accessible at run time be included in the package." I think we're stuck with this, other than a few wild ideas like compiling the schemas directly into Python code (which still strikes me as a good idea!).

Reply:
Yes, you have to put them in the package if you are using pkg_resources. This is the same reason why I had to relocate the astro_metadata_translator corrections YAML files into the python directory.

Reply:
Well, that's sad!

This is fine for the short term. Longer term, we should think about how we actually want to distribute these schema files — there should be one canonical place for them, and it's not _obvious_ that should be coupled to Python software releases. But let's not block on that now.